src/libs/antares/study/scenario-builder/rules.h

```diff
@@ -112,7 +112,7 @@ class Rules final : private Yuni::NonCopyable<Rules>
     //! Wind
     windTSNumberData wind;
 
-    // gp : change these 2 arrays into std containers, like std::vector 
+    // gp : change these 3 arrays into std containers, like std::vector 
```

Comment:
That would be nice

Reply:
I do it with **std::vector**. I would have liked to do it with **std::array** but I don't see how to initialize the arrays size at declaration, as it seems to be mandatory.